libsrc/runtime/aslax7.s

```diff
@@ -0,0 +1,21 @@
+;
+; Miloslaw Smyk, 2024
+;
+; CC65 runtime: Scale the primary register by 128, unsigned
+;
+
+        .export         shlax7, aslax7
+
+aslax7:
```

Comment:
are you sure aslax7 shouldn't preserve the sign?

Reply:
Hi!
Yes. I was surprised about it too at first when looking at the other aslax*, but that's normal: https://en.wikipedia.org/wiki/Arithmetic_shift